presto-main/src/main/java/com/facebook/presto/operator/aggregation/DoubleHistogramAggregation.java

```diff
@@ -33,15 +33,15 @@
 import static com.facebook.presto.util.Failures.checkCondition;
 
 @AggregationFunction("numeric_histogram")
```

Comment:
This name was given already before. It worked on DOUBLE, so any type coercible to DOUBLE would benefit from this function.